api/admin/controller/__init__.py

```diff
@@ -2544,11 +2544,10 @@ def validate_number(self, settings):
         2) positive, and 3) lower than the specified maximum, if there is one."""
         # Find the fields that should have numeric input and are not blank.
         number_fields = filter(
-                            lambda s: ("number" in [s.get("type"), s.get("format")]) and
+                            lambda s: s.get("type") == "number" and
                             (flask.request.form.get(s.get("key")) or flask.request.form.get("value"))
                             , settings
                         )
```

Comment:
I don't have an opinion on this but I want to make sure @adriana-alter sees this and confirms it's all right.

Reply:
Adriana told me that this was actually her change and you just merged it in.